src/main/java/org/jenkinsci/plugins/GithubRequireOrganizationMembershipACL.java

```diff
@@ -239,6 +250,14 @@ private boolean checkReadPermission(@NonNull Permission permission) {
                 || id.equals("hudson.model.Item.Read"));
     }
 
+    private boolean checkAgentUserPermission(@NonNull Permission permission) {
+        String id = permission.getId();
+        return (id.equals("hudson.model.Hudson.Read")
+                || id.equals("hudson.model.Computer.Create")
+                || id.equals("hudson.model.Computer.Connect")
+                || id.equals("hudson.model.Computer.Configure"));
```

Comment:
I would recommend taking the type-safe approach, in order to avoid typos, and also to allow the compiler to let us know if any of these permissions are removed/renamed.

Here is what I used in my duplicate PR:
https://github.com/jenkinsci/github-oauth-plugin/pull/246/files#diff-cc7ab75bded9621aa042633b3d55ab401e4141ae1ab84e22b3b1a00dac737148R262-R267